lib/rspec/matchers/built_in/contain_exactly.rb

```diff
@@ -154,7 +154,7 @@ def unmatched_item_count
             def +(derived_candidate_solution)
               self.class.new(
                 unmatched_expected_indexes + derived_candidate_solution.unmatched_expected_indexes,
-                unmatched_actual_indexes   + derived_candidate_solution.unmatched_actual_indexes,
+                unmatched_actual_indexes + derived_candidate_solution.unmatched_actual_indexes,
```

Comment:
Personally I'd rather this remain aligned for asthetic reasons, is it possible to disable the cop, what do others think?